openquake/commonlib/oqvalidation.py

```diff
@@ -838,6 +839,12 @@
                    'event_based_damage',
                    'scenario_damage']
 
+LOSS_TYPES = [
+    'structural', 'nonstructural', 'contents', 'business_interruption']
+LOSS_TYPE_COMBINATIONS = [
+    '+'.join(s) for l_idx in range(len(LOSS_TYPES))
+    for s in itertools.combinations(LOSS_TYPES, l_idx + 1)]
+
```

Comment:
Please change the name from LOSS_TYPES to COST_TYPES.

Reply:
Done